[BUG]: Django project pre-commit pylint failing on django imports

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

-   imAsparky cookiecutter-pypackage version:
-   Python version:
-   OS: [e.g. iOS]
-   Browser [e.g. chrome, safari]
-   Version [e.g. 22]

**Smartphone (please complete the following information):**

-   Device: [e.g. iPhone6]
-   OS: [e.g. iOS8.1]
-   Browser [e.g. stock browser, safari]
-   Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

pylint...................................................................Failed
- hook id: pylint
- exit code: 30

************* Module users.admin
users/admin.py:3:0: E0401: Unable to import 'django.contrib' (import-error)
users/admin.py:4:0: E0401: Unable to import 'django.contrib.auth.admin' (import-error)
users/admin.py:5:0: E0401: Unable to import 'django.contrib.auth.decorators' (import-error)
users/admin.py:21:0: R0903: Too few public methods (0/2) (too-few-public-methods)
************* Module django_boilerplate.urls
django_boilerplate/urls.py:16:0: E0401: Unable to import 'django.conf' (import-error)
django_boilerplate/urls.py:17:0: E0401: Unable to import 'django.contrib' (import-error)
django_boilerplate/urls.py:18:0: E0401: Unable to import 'django.urls' (import-error)
django_boilerplate/urls.py:19:0: E0401: Unable to import 'allauth' (import-error)
django_boilerplate/urls.py:30:4: E0401: Unable to import 'debug_toolbar' (import-error)

--- a/tests/test_bake_django.py
+++ b/tests/test_bake_django.py
@@ -778,7 +778,7 @@ def test_baked_django_base_settings_base_file_ok(cookies):
     assert 'ROOT_URLCONF = "django_boilerplate.urls"' in settings_file
     assert 'WSGI_APPLICATION = "django_boilerplate.wsgi.application"' in settings_file
     assert 'LANGUAGE_CODE = "en"' in settings_file
-    assert "    ('en', _(\"English\"))," in settings_file
+    assert "    (\"en\", _(\"English\"))," in settings_file
     assert 'TIME_ZONE = "UTC"' in settings_file
     assert "USE_I18N = True" in settings_file
 

--- a/{{cookiecutter.git_project_name}}/.env/.local
+++ b/{{cookiecutter.git_project_name}}/.env/.local
@@ -13,6 +13,3 @@ DB_USER=""
 DB_PASSWORD=""
 DB_HOST=""
 DB_PORT=""
-
-
-

--- a/{{cookiecutter.git_project_name}}/.env/.testing
+++ b/{{cookiecutter.git_project_name}}/.env/.testing
@@ -15,4 +15,3 @@ INTERNAL_IPS=""
 # If using other, a DB_URL connection string must be supplied.
 TESTING_DATABASE=sqlite3
 DB_URL=""
-

--- a/{{cookiecutter.git_project_name}}/.github/workflows/test_contribution.yaml
+++ b/{{cookiecutter.git_project_name}}/.github/workflows/test_contribution.yaml
@@ -13,7 +13,7 @@ jobs:
     test_contribs:
         strategy:
             matrix:
-                python-version: ["3.6", "3.7", "3.8", "3.9"]
+                python-version: ["3.8", "3.9", "3.10"]
                 os: [macos-latest, ubuntu-latest, windows-latest]
 
         runs-on: ${{ matrix.os }}

--- a/{{cookiecutter.git_project_name}}/.pre-commit-config.yaml
+++ b/{{cookiecutter.git_project_name}}/.pre-commit-config.yaml
@@ -1,7 +1,8 @@
+default_language_version:
+  python: python3.10
 default_stages: [commit]
 fail_fast: true
 exclude: ".conf.py|^tests/|migrations|.git|.tox"
-
 ci:
   autofix_commit_msg: |
     chore(pre-commit): Auto fixes from pre-commit.com hooks
@@ -40,11 +41,6 @@ repos:
       - id: requirements-txt-fixer
       - id: check-merge-conflict
 
-  - repo: https://github.com/pycqa/pylint
-    rev: v2.10.2
-    hooks:
-      - id: pylint
-
   - repo: https://github.com/psf/black
     rev: 21.8b0
     hooks:
@@ -63,7 +59,6 @@ repos:
     hooks:
       - id: flake8
         additional_dependencies: [flake8-typing-imports==1.10.0]
-        # exclude:
 
   - repo: https://github.com/PyCQA/isort
     rev: 5.9.3

--- a/{{cookiecutter.git_project_name}}/LICENSE.rst
+++ b/{{cookiecutter.git_project_name}}/LICENSE.rst
@@ -22,8 +22,7 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-{% elif cookiecutter.open_source_license == 'BSD license' %}
+SOFTWARE.{% elif cookiecutter.open_source_license == 'BSD license' %}
 .. _bsd-license:
 ===========
 BSD License
@@ -55,8 +54,7 @@ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
 SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCEOR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
-THIS SOFTWARE, EVEN IF ADVISEDOF THE POSSIBILITY OF SUCH DAMAGE.
-{% elif cookiecutter.open_source_license == 'ISC license' -%}
+THIS SOFTWARE, EVEN IF ADVISEDOF THE POSSIBILITY OF SUCH DAMAGE.{% elif cookiecutter.open_source_license == 'ISC license' -%}
 .. _isc-license:
 ===========
 ISC License
@@ -74,8 +72,7 @@ FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
 INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
 LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
 OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
-PERFORMANCE OF THIS SOFTWARE.
-{% elif cookiecutter.open_source_license == 'Apache Software License 2.0' -%}
+PERFORMANCE OF THIS SOFTWARE.{% elif cookiecutter.open_source_license == 'Apache Software License 2.0' -%}
 .. _apache-license:
 ===========================
 Apache Software License 2.0
@@ -93,8 +90,7 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
-{% elif cookiecutter.open_source_license == 'GNU General Public License v3' -%}
+limitations under the License.{% elif cookiecutter.open_source_license == 'GNU General Public License v3' -%}
 .. _gnu-license:
 ==========================
 GNU GENERAL PUBLIC LICENSE
@@ -129,5 +125,4 @@ into proprietary programs.  If your program is a subroutine library, you
 may consider it more useful to permit linking proprietary applications with
 the library.  If this is what you want to do, use the GNU Lesser General
 Public License instead of this License.  But first, please read
-<http://www.gnu.org/philosophy/why-not-lgpl.html>.
-{% endif %}
+<http://www.gnu.org/philosophy/why-not-lgpl.html>.{% endif %}

--- a/{{cookiecutter.git_project_name}}/compose/docker-compose-swarm.yml
+++ b/{{cookiecutter.git_project_name}}/compose/docker-compose-swarm.yml
@@ -59,5 +59,3 @@ secrets:
   #   external: true
   # db_port:
   #   external: true
-
-

--- a/{{cookiecutter.git_project_name}}/config/settings/base.py
+++ b/{{cookiecutter.git_project_name}}/config/settings/base.py
@@ -9,9 +9,11 @@ For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
 
-from pathlib import Path
 import os
+from pathlib import Path
+
 from django.utils.translation import gettext_lazy as _
+
 from .username_blacklist import data as username_blacklist
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
@@ -21,7 +23,6 @@ BASE_DIR = Path(__file__).resolve().parent.parent.parent
 
 INSTALLED_APPS = [
     "users",
-
     "django.contrib.admin",
     "django.contrib.auth",
     "django.contrib.contenttypes",
@@ -29,14 +30,11 @@ INSTALLED_APPS = [
     "django.contrib.sites",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-
     "allauth",
     "allauth.account",
     "allauth.socialaccount",
-
-    'allauth.socialaccount.providers.github',
-    'allauth.socialaccount.providers.google',
-
+    "allauth.socialaccount.providers.github",
+    "allauth.socialaccount.providers.google",
 ]
 
 AUTH_USER_MODEL = "users.CustomUser"
@@ -48,7 +46,7 @@ SITE_ID = {{cookiecutter.SITE_ID}}
 
 # LOGIN_REDIRECT_URL For new project convenience, change to your project requirements.
 LOGIN_REDIRECT_URL = "/admin/"
-LOGOUT_REDIRECT_URL = '/accounts/login/'
+LOGOUT_REDIRECT_URL = "/accounts/login/"
 
 
 MIDDLEWARE = [
@@ -76,19 +74,16 @@ TEMPLATES = [
                 "django.contrib.auth.context_processors.auth",
                 "django.contrib.messages.context_processors.messages",
                 "django.template.context_processors.request",
-
             ],
         },
     },
 ]
 
 AUTHENTICATION_BACKENDS = [
-
     # Needed to login by username in Django admin, regardless of `allauth`
-    'django.contrib.auth.backends.ModelBackend',
-
+    "django.contrib.auth.backends.ModelBackend",
     # `allauth` specific authentication methods, such as login by e-mail
-    'allauth.account.auth_backends.AuthenticationBackend',
+    "allauth.account.auth_backends.AuthenticationBackend",
 ]
 
 WSGI_APPLICATION = "{{ cookiecutter.project_slug}}.wsgi.application"
@@ -132,7 +127,7 @@ LANGUAGE_CODE = "{{cookiecutter.LANGUAGE_CODE}}"
 {%- with languages = cookiecutter.LANGUAGES.replace(' ', '').split(',') %}
 
 LANGUAGES = [{% for language in languages %}
-    ('{{ language }}', _("{{ language_labels[language] }}")),{% endfor %}
+    ("{{ language }}", _("{{ language_labels[language] }}")),{% endfor %}
 ]
 {% endwith %}
 
@@ -162,13 +157,14 @@ DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
 # Django Allauth Settings
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
-ACCOUNT_AUTHENTICATION_METHOD = "username_email" # Default dj-allauth == username
-ACCOUNT_EMAIL_REQUIRED = True                    # Default dj-allauth == False
-ACCOUNT_UNIQUE_EMAIL = True                      # Default dj-allauth
-ACCOUNT_EMAIL_VERIFICATION = 'mandatory'         # Default dj-allauth (optional)
-ACCOUNT_EMAIL_CONFIRMATION_EXPIRE_DAYS = 3       # Default dj-allauth
-ACCOUNT_LOGIN_ATTEMPTS_LIMIT = 5                 # Default dj-allauth
-ACCOUNT_USERNAME_REQUIRED = True                 # Default dj-allauth
-ACCOUNT_USERNAME_MIN_LENGTH = 3                  # Default dj-allauth == 1
+ACCOUNT_AUTHENTICATION_METHOD = (
+    "username_email"  # Default dj-allauth == username
+)
+ACCOUNT_EMAIL_REQUIRED = True  # Default dj-allauth == False
+ACCOUNT_UNIQUE_EMAIL = True  # Default dj-allauth
+ACCOUNT_EMAIL_VERIFICATION = "mandatory"  # Default dj-allauth (optional)
+ACCOUNT_EMAIL_CONFIRMATION_EXPIRE_DAYS = 3  # Default dj-allauth
+ACCOUNT_LOGIN_ATTEMPTS_LIMIT = 5  # Default dj-allauth
+ACCOUNT_USERNAME_REQUIRED = True  # Default dj-allauth
+ACCOUNT_USERNAME_MIN_LENGTH = 3  # Default dj-allauth == 1
 ACCOUNT_USERNAME_BLACKLIST = username_blacklist
-

--- a/{{cookiecutter.git_project_name}}/config/settings/local.py
+++ b/{{cookiecutter.git_project_name}}/config/settings/local.py
@@ -1,14 +1,15 @@
 """Django local settings for {{cookiecutter.git_project_name}} project."""
 
-from .base import *  # noqa
 import environ
+
+from .base import *  # noqa: F405 F401 F403
 
 # Read from environment variables file
 env = environ.Env(
     # set casting, default value
     DEBUG=(bool, False)
 )
-environ.Env.read_env(os.path.join(BASE_DIR, ".env/.local"))
+environ.Env.read_env(os.path.join(BASE_DIR, ".env/.local"))  # noqa: F405
 
 SECRET_KEY = env("DJANGO_SECRET_KEY")
 
@@ -60,6 +61,6 @@ DEBUG_TOOLBAR_CONFIG = {
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "NAME": BASE_DIR / "db.sqlite3",  # noqa: F405
     }
 }

--- a/{{cookiecutter.git_project_name}}/config/settings/production.py
+++ b/{{cookiecutter.git_project_name}}/config/settings/production.py
@@ -1,8 +1,9 @@
 """Django production settings for {{cookiecutter.git_project_name}} project."""
 
-from .base import *  # noqa
-from django.conf import settings
 import environ
+from django.conf import settings
+
+from .base import *  # noqa: F405 F401 F403
 {% if cookiecutter.deploy_with_docker == "swarm" %}
 env = environ.FileAwareEnv()
 {% else %}
@@ -10,7 +11,7 @@ env = environ.Env(
     # set casting, default value
     DEBUG=(bool, False)
 )
-environ.Env.read_env(os.path.join(BASE_DIR, ".env/.staging"))
+environ.Env.read_env(os.path.join(BASE_DIR, ".env/.production"))  # noqa: F405
 {% endif %}
 # fmt: off
 SECRET_KEY = env('DJANGO_SECRET_KEY', default="!!!INSECURE_PRODUCTION_SECRET!!!")
@@ -29,9 +30,9 @@ EMAIL_BACKEND = env(
 DATABASES = {"default": env.db()}
 
 # Check it is safe to run in production.
-assert (
+assert (  # nosec
     not settings.DEBUG
-), "DEBUG mode should be False for the production environment."  # nosec
-assert (
+), "DEBUG mode should be False for the production environment."
+assert (  # nosec
     env("DJANGO_SECRET_KEY") != "!!!INSECURE_PRODUCTION_SECRET!!!"
-), "The DJANGO_SECRET_KEY must be set for production."  # nosec
+), "The DJANGO_SECRET_KEY must be set for production."

--- a/{{cookiecutter.git_project_name}}/config/settings/staging.py
+++ b/{{cookiecutter.git_project_name}}/config/settings/staging.py
@@ -1,8 +1,9 @@
 """Django staging settings for {{cookiecutter.git_project_name}} project."""
 
-from .base import *  # noqa
-from django.conf import settings
 import environ
+from django.conf import settings
+
+from .base import *  # noqa: F405 F401 F403
 {% if cookiecutter.deploy_with_docker == "swarm" %}
 env = environ.FileAwareEnv()
 {% else %}
@@ -10,7 +11,7 @@ env = environ.Env(
     # set casting, default value
     DEBUG=(bool, False)
 )
-environ.Env.read_env(os.path.join(BASE_DIR, ".env/.staging"))
+environ.Env.read_env(os.path.join(BASE_DIR, ".env/.staging"))  # noqa: F405
 {% endif %}
 # fmt: off
 SECRET_KEY = env('DJANGO_SECRET_KEY', default="!!!INSECURE_PRODUCTION_SECRET!!!")
@@ -28,10 +29,10 @@ EMAIL_BACKEND = env(
 
 DATABASES = {"default": env.db()}
 
-# Check it is safe to run in producion.
-assert (
+# Check it is safe to run in production.
+assert (  # nosec
     not settings.DEBUG
-), "DEBUG mode should be off for the production environment."  # nosec
-assert (
+), "DEBUG mode should be False for the production environment."
+assert (  # nosec
     env("DJANGO_SECRET_KEY") != "!!!INSECURE_PRODUCTION_SECRET!!!"
-), "The DJANGO_SECRET_KEY must be set for production."  # nosec
+), "The DJANGO_SECRET_KEY must be set for production."

--- a/{{cookiecutter.git_project_name}}/config/settings/test.py
+++ b/{{cookiecutter.git_project_name}}/config/settings/test.py
@@ -1,15 +1,16 @@
 """Django test settings for {{cookiecutter.git_project_name}} project."""
 
-from .base import *  # noqa
-from django.conf import settings
 import environ
+from django.conf import settings
+
+from .base import *  # noqa: F405 F401 F403
 
 # Read from environment variables file
 env = environ.Env(
     # set casting, default value
     DEBUG=(bool, False)
 )
-environ.Env.read_env(os.path.join(BASE_DIR, ".env/.testing"))
+environ.Env.read_env(os.path.join(BASE_DIR, ".env/.testing"))  # noqa: F405
 
 SECRET_KEY = env("DJANGO_SECRET_KEY")
 

--- a/{{cookiecutter.git_project_name}}/requirements_dev.txt
+++ b/{{cookiecutter.git_project_name}}/requirements_dev.txt
@@ -1,7 +1,4 @@
 
 -r config/requirements/local.txt
--r config/requirements/test.txt
-{% if cookiecutter.include_sphinx_docs == "y" %}
--r docs/requirements.txt
-{% endif %}
-
+-r config/requirements/test.txt{% if cookiecutter.include_sphinx_docs == "y" %}
+-r docs/requirements.txt{% endif %}

--- a/{{cookiecutter.git_project_name}}/templates/account/password_reset_done.html
+++ b/{{cookiecutter.git_project_name}}/templates/account/password_reset_done.html
@@ -7,10 +7,10 @@
 
 {% block content %}
     <h1>{% trans "Password Reset" %}</h1>
-    
+
     {% if user.is_authenticated %}
     {% include "account/snippets/already_logged_in.html" %}
     {% endif %}
-    
+
     <p>{% blocktrans %}We have sent you an e-mail. Please contact us if you do not receive it within a few minutes.{% endblocktrans %}</p>
 {% endblock %}

--- a/{{cookiecutter.git_project_name}}/tox.ini
+++ b/{{cookiecutter.git_project_name}}/tox.ini
@@ -79,3 +79,19 @@ commands =
 
 [pydocstyle]
 ignore = D213
+
+[flake8]
+    ; ignore = Add codes to ignore here.
+    max-line-length = 110
+    max-complexity = 10
+    select = B,C,E,F,W,T4,B9
+
+; Black default formats code line length to 88.  Black doesnt take into account
+; text and URL.  Flake max-line-length is 110 to allow for these odd occaisons.
+
+; Examples of flake8 codes.
+; E203 	whitespace before ‘,’, ‘;’, or ‘:’
+; E266 	too many leading ‘#’ for block comment
+; E501 (^) 	line too long (89 > 88 characters)  max-line-length setting.
+; W503 (*) 	line break before binary operator
+; F403 ‘from module import *’ used; unable to detect undefined names

--- a/{{cookiecutter.git_project_name}}/users/admin.py
+++ b/{{cookiecutter.git_project_name}}/users/admin.py
@@ -4,7 +4,7 @@ from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
 from django.contrib.auth.decorators import login_required
 
-from .forms import CustomUserCreationForm, CustomUserChangeForm
+from .forms import CustomUserChangeForm, CustomUserCreationForm
 from .models import CustomUser
 
 # Customise the Admin title and header.

--- a/{{cookiecutter.git_project_name}}/users/apps.py
+++ b/{{cookiecutter.git_project_name}}/users/apps.py
@@ -1,6 +1,9 @@
+"""Custom Users APP configuration file"""
 from django.apps import AppConfig
 
 
 class UsersConfig(AppConfig):
-    default_auto_field = 'django.db.models.BigAutoField'
-    name = 'users'
+    """Custom Users Configuration class"""
+
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "users"

--- a/{{cookiecutter.git_project_name}}/users/apps.py
+++ b/{{cookiecutter.git_project_name}}/users/apps.py
@@ -1,8 +1,9 @@
-"""Custom Users APP configuration file"""
+"""Custom Users APP configuration file."""
 from django.apps import AppConfig
 
 
 class UsersConfig(AppConfig):
+
     """Custom Users Configuration class"""
 
     default_auto_field = "django.db.models.BigAutoField"

--- a/{{cookiecutter.git_project_name}}/users/forms.py
+++ b/{{cookiecutter.git_project_name}}/users/forms.py
@@ -1,7 +1,8 @@
 """{{cookiecutter.git_project_name}} project CustomUser Forms."""
 
-from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+from django.contrib.auth.forms import UserChangeForm, UserCreationForm
 from django.utils.translation import gettext_lazy as _
+
 from .models import CustomUser
 
 
@@ -14,7 +15,9 @@ class CustomUserCreationForm(UserCreationForm):
 
         error_messages = {
             "username": {
-                "unique": _("That user name already exists, please choose another.")
+                "unique": _(
+                    "That user name already exists, please choose another."
+                )
             }
         }
 

--- a/{{cookiecutter.git_project_name}}/users/models.py
+++ b/{{cookiecutter.git_project_name}}/users/models.py
@@ -2,17 +2,19 @@
 
 from django.apps import apps
 from django.contrib import auth
+from django.contrib.auth.base_user import BaseUserManager
 from django.contrib.auth.hashers import make_password
 from django.contrib.auth.models import AbstractUser
-from django.contrib.auth.base_user import BaseUserManager
-from django.utils.translation import gettext_lazy as _
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 
 class CustomUserManager(BaseUserManager):
     use_in_migrations = True
 
-    def _create_user(self, username, email, password, **extra_fields): # pragma: no cover
+    def _create_user(
+        self, username, email, password, **extra_fields
+    ):  # pragma: no cover
         """
         Create and save a user with the given username, email, and password.
         """
@@ -36,7 +38,9 @@ class CustomUserManager(BaseUserManager):
 
         return user
 
-    def create_user(self, username, email=None, password=None, **extra_fields): # pragma: no cover
+    def create_user(
+        self, username, email=None, password=None, **extra_fields
+    ):  # pragma: no cover
         """
         Create and save a standard user with the supplied username, email, and password.
         """
@@ -45,7 +49,9 @@ class CustomUserManager(BaseUserManager):
         extra_fields.setdefault("is_superuser", False)
         return self._create_user(username, email, password, **extra_fields)
 
-    def create_superuser(self, username, email=None, password=None, **extra_fields): # pragma: no cover
+    def create_superuser(
+        self, username, email=None, password=None, **extra_fields
+    ):  # pragma: no cover
         """
         Create and save a super user with the supplied username, email, and password.
         """
@@ -66,8 +72,13 @@ class CustomUserManager(BaseUserManager):
         return self._create_user(username, email, password, **extra_fields)
 
     def with_perm(
-        self, perm, is_active=True, include_superusers=True, backend=None, obj=None
-    ): # pragma: no cover
+        self,
+        perm,
+        is_active=True,
+        include_superusers=True,
+        backend=None,
+        obj=None,
+    ):  # pragma: no cover
         if backend is None:
             backends = auth._get_backends(return_tuples=True)
             if len(backends) == 1:
@@ -79,7 +90,8 @@ class CustomUserManager(BaseUserManager):
                 )
         elif not isinstance(backend, str):
             raise TypeError(
-                "backend must be a dotted import path string (got %r)." % backend
+                "backend must be a dotted import path string (got %r)."
+                % backend
             )
         else:
             backend = auth.load_backend(backend)

--- a/{{cookiecutter.git_project_name}}/{{cookiecutter.project_slug}}/urls.py
+++ b/{{cookiecutter.git_project_name}}/{{cookiecutter.project_slug}}/urls.py
@@ -13,10 +13,10 @@ Including another URLconf
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from allauth import account
 from django.conf import settings
 from django.contrib import admin
 from django.urls import include, path
-from allauth import account
 
 urlpatterns = [
     path("admin/", admin.site.urls),

